client/src/components/new-request/search.vue

```diff
@@ -10,7 +10,7 @@
     </v-row>
 
     <v-row class="mt-4" no-gutters>
-      <v-col cols="5">
+      <v-col cols="4">
```

Comment:
Just curious here, but does going down to 4 columns cut off a few of verbiage after selecting an item?
For example, does `Change the name of an existing`.. selection still fit in the box after selection? 
I ask only because if it doesn't, UX might want to add a tooltip for any selections that are not fully visible just like the `Change (alter) business type of a`.. selection. 
Just a heads up!

Reply:
Yeah I did this so that 'Select a Jurisdiction' wouldn't be cut off, which is specified in the ticket. Unfortunately I can't set the middle column to 3 and have the other 2 on either side fit properly. For now I set them all to 4, but we will see if UX will let this by - I may have to set the width differently

Reply:
Sounds good!